module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -377,9 +376,10 @@ public function formatEmailMessage(array $requestParams = [])
                     ?? $this->vufindConfig['Site']['displayDateFormat'] ?? 'Y-m-d';
                 $value = date($format, strtotime($value));
             }
-
-            $label = isset($el['label']) ? $this->translate($el['label']) : null;
-            $params[] = ['type' => $type, 'value' => $value, 'label' => $label];
+            $el['value'] = $value;
+            $el['label'] = isset($el['label']) ? $this->translate($el['label'])
+                : null;
+            $params[] = $el;
```

Comment:
Would it be cleaner to implement this as below?

<pre>
            $label = isset($el['label']) ? $this->translate($el['label'])
                : null;
            $params[] = $el + compact('value', 'label');
</pre>

I realize it probably doesn't hurt anything, but it seems wrong to manipulate the actual element configuration in order to pass it to the template... feels more appropriate to merge it with additional fields. Though feel free to disagree if I'm missing a reason for doing it as-is.